docs/plugins/jenkins.zh.md

```diff
@@ -46,7 +46,18 @@ chown -R 1000:1000 <your-dtm-home-dir>/data/jenkins-volume/
 --8<-- "jenkins.yaml"
 ```
 
-当前，所有配置项均为必填。
+#### 默认配置
+
+| key | default value | description |
+|  ----  | ----  | ----  |
+| chart.chart_name | jenkins/jenkins | helm 包名称 |
+| chart.timeout | 10m | 等待部署成功的时间 |
```

Comment:
The default value in the yaml file is 5 minutes, but the code is set to 10 minutes.